src/aircraft_cmd.cpp

```diff
@@ -463,6 +463,14 @@ Money Aircraft::GetRunningCost() const
 {
 	const Engine *e = this->GetEngine();
 	uint cost_factor = GetVehicleProperty(this, PROP_AIRCRAFT_RUNNING_COST_FACTOR, e->u.air.running_cost);
+
+	/* running costs if in depot -- I haven't gotten this one working yet, can't figure out how to tell if plane is in the depot */
+	if (this->current_order.IsType(OT_GOTO_DEPOT)) cost_factor /= _settings_game.difficulty.vehicle_costs_in_depot;
+
+	/* running costs if stopped */
+	if ((this->cur_speed == 0) && !(this->current_order.IsType(OT_GOTO_DEPOT))) cost_factor /= _settings_game.difficulty.vehicle_costs_when_stopped;
```

Comment:
Suggestion: if you use `else` here the inverted condition is not necessary, and I believe the code should function identically.